### ISSUE TITLE
Can I use the Same Login and Password Combo for Different URLs with the same Domain?

### ISSUE THREAD

**Issue by CerebralFreeze** [keepassxreboot/keepassxc#4899]
## Summary
Hiya' Folks!  I hope this is isn't too stupid of a question / feature request.  Previously, it seemed that I could use a login and password for any website as long as the domain name was the same.  But, I must have chanced a setting or something.  

If the full URL of the website that I'm trying to gain access to is not the same exact URL in KeePassXC's URL entry field, KeePassXC's browser extension doesn't recognize the website.

I have tried to add multiple variations of the URL to the Browser Integration section of the edit entry but that doesn't seem to help.

Someone please advise.  Thank you, sir or madam!

## Examples
If one of the password entries contains login.chase.com but I am at chase.com, it seems that the program doesn't recognize it.  It seems like it did before.  Maybe I'm starting to go mentally crazy from the Corona Virus.

## Context
This is important because I don't want to make multiple password entries for each one of my accounts because there are multiple URL's I can use to access a website.

**Comment by varjolintu** [comment]
If you use `https://chase.com` as the entry URL, it will work as a wildcard for `https://*.chase.com`. `https://login.chase.com` would only work with URL's starting with `https://login.chase.com/`.

**Comment by CerebralFreeze** [comment]
Varjolintu,

That's what I thought!  But, for some reason, it's not working anymore.  Maybe I touched a setting that messed with it.  

This past week, I've been fooling around with the settings and nothing seemed to work.  I'm not sure if it's me or the program.  I've always had the "Save only the domain" check-marked but obviously, that isn't the issue.

I've just unchecked the option, "Return only best-matching credentials" and that _might_ have done it.  I'll let everyone know in about a week of use if that's the solution.  If it is the solution, the developers should change the option to say something like "Allow the use of credentials that share only the domain with the accessing website".  
 
I'll try to fool around with the settings and see it fixes the problem and then provide an update for everyone.
